.ci/deps.sh

```diff
@@ -3,22 +3,24 @@ set -x
 
 # Choose the python versions to install deps for
 case $CIRCLE_NODE_INDEX in
- 0) dep_versions=( "3.3.6" "3.4.3" "3.5.1" ) ;;
+ 0) dep_versions=( "3.3.6" "3.4.3" "3.5.1" "2.7.10" ) ;;
  1) dep_versions=( "3.4.3" ) ;;
  2) dep_versions=( "3.3.6" ) ;;
- *) dep_versions=( "3.5.1" ) ;;
+ *) dep_versions=( "3.5.1" "2.7.10" ) ;;
```

Comment:
What i tried to say in gitter is it should be added at https://github.com/coala-analyzer/coala-bears/pull/229/files#diff-5c64bb36a9fa177d5dfeb57b93e57a6cR49